nixos/tests/sudo.nix

```diff
@@ -0,0 +1,15 @@
+import ./make-test.nix {
+  name = "sudo";
+  nodes = {
+  machine = { pkgs, config, ... }:
+    {
+      security.sudo.enable = true;
+    };
+  };
+  testScript = ''
+    startAll;
+    $machine->waitForUnit("multi-user.target");
+    $machine->succeed("sudo echo foobar");
```

Comment:
You probably want to test whether sudo works for non-root users, so something like `su - alice -c 'sudo ...'`.